__tests__/app.test.js

```diff
@@ -81,3 +81,32 @@ describe("GET /api/reviews/:review_id", () => {
             });
     });
 });
+
+describe("GET /api/reviews", () => {
+    test("200: Responds with an array of review objects", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body }) => {
+                const { reviews } = body;
+                reviews.forEach((review) => {
```

Comment:
When using .forEach, we need to assert against the array length first. Otherwise, an empty array will skip the .forEach and the test will automatically pass